lua/refactoring/tasks/indent.lua

```diff
@@ -0,0 +1,68 @@
+local Point = require("refactoring.point")
+local Region = require("refactoring.region")
+
+local function indent_block(start_row, end_row, indent_amount)
+    -- TODO: have to be a better way to do this
```

Comment:
i agree but lsp client text_edits are applied next event loop.  So you have to execute these things afterwords :(

Just looking at this, it may be better to apply the indents to the raw text than it is to the code post application (if I have understood what this code does correctly).

Reply:
Gotcha, I'll move to the raw text then in 106 itself.